docs/sources/developers/plugins/_index.md

```diff
@@ -6,77 +6,48 @@ title: Build a plugin
 weight: 200
 ---
 
-# Build a plugin
+# Welcome to the Grafana plugin developer's guide
```

Comment:
I scanned through our other docs and didn't see another page title with "Welcome to the..." in it. Maybe there is another title that would work like "Grafana plugin developer's guide"?

Reply:
Oh gee. Honestly, my preference would be for us to add "Welcome to the... to all the other doc pages. Better for Grafana's personality, I think.

Until that happens, I think you speak wisely.